flink-runtime/src/main/java/org/apache/flink/runtime/resourcemanager/StandaloneResourceManagerFactory.java

```diff
@@ -74,4 +81,20 @@
 			standaloneClusterStartupPeriodTime,
 			AkkaUtils.getTimeoutAsTime(configuration));
 	}
+
+	/**
+	 * Get the configuration for standalone ResourceManager, overwrite invalid configs.
+	 *
+	 * @param configuration configuration object
+	 * @return the configuration for standalone ResourceManager
+	 */
+	private static Configuration getConfigurationForStandaloneResourceManager(Configuration configuration) {
```

Comment:
```suggestion
	private static Configuration removeMaxSlotNumberIfSet(Configuration configuration) {
```
As this is the only thing which is done here now, I would suggest to be more explicit.